### PR TITLE
Bar method add

### PR DESCRIPTION
**Pull request by tylndnzplt**
Bar method add

### CODE REVIEW

File: DummyConsoleApp/Program.cs [hunk 1]
```diff
@@ -15,5 +15,10 @@ static void Foo()
         {
             Console.WriteLine("Hello!");
         }
+
+        static void Bar()
```
Comment (tuncdan) [inline DummyConsoleApp/Program.cs:19]: Dummy work!

**Review by tuncdan — CHANGES_REQUESTED** [review]
Add Tar!

**Comment by tylndnzplt** [comment]
tar will add

File: DummyConsoleApp/Program.cs [hunk 2]
```diff
@@ -20,5 +20,10 @@ static void Bar()
         {
             Console.WriteLine("Bar");
         }
+
+        static void Tar()
```
Comment (tuncdan) [inline DummyConsoleApp/Program.cs:24]: Dummy Tar!

**Review by tuncdan — APPROVED** [review]

**Comment by tylndnzplt** [comment]
done

**Pull request merged**